model.js

```diff
@@ -17,4 +15,17 @@ readEndpointsFile = async () => {
         return parsedFile;
 }
 
-module.exports = { selectTopics, readEndpointsFile };
\ No newline at end of file
+selectArticleById = async (article_id) => {
+    const result = await db.query('SELECT * FROM articles WHERE article_id = $1;', [article_id])
+    const article = result.rows[0]
+    if (!article) {
```

Comment:
I'd be a bit more specific here and check the length of the rows which come back to make sure it's 0.
As what is returned should still be an array.